deploy/deploy.sh

```diff
@@ -136,8 +137,8 @@ oc wait -n $NS --for="condition=Ready" pod -l app=workload-web-app --timeout=120
 if [[ ! -z "${GRAFANA_DASHBOARD}" ]]; then
   echo "Creating Grafana Dashboard for the app"
   if [[ -z "${RHOAM}" ]]; then
-    oc apply -n $NS -f $DIR/dashboard.yaml
+    oc apply -n $OBSERVABILITY_NS -f $DIR/dashboard.yaml
```

Comment:
Is this change needed for RHMI out of curiousity ? :thinking:

Reply:
very good question and catch , we don't need this for RHMI